i18n/sv.toml

```diff
@@ -157,3 +157,6 @@ other = "Skillnader mot ACME-RFC:n"
 
 [onboarding_customers]
 other = "Skola in dina användare med Let's Encrypt och ACME"
+
+[paypal_donate_alt]
+other = "Donera med PayPal-knappen"
```

Comment:
"knappen" is definitive form. The English version says just "Donate with PayPal button" (without "the") so I recommend keeping the alt text "Donera med PayPal-knapp" that I chose in `content/sv/donate.html`.

Reply:
I see this is taken from the official Swedish PayPal translation but I think it's not so good...

Reply:
I'm conflicted in that point. I'll think about it while I wait for @bdaehlie answers on https://github.com/letsencrypt/website/pull/809#issuecomment-568291536

Reply:
I've updated [paypal_donate_alt] with your version. If you want to translate [paypal_donate_title], feel free to open a PR :-)